frontend-react/cypress/support/commands/organizations.js

```diff
@@ -0,0 +1,29 @@
+import { GLOBAL_STORAGE_KEYS } from "../../../src/utils/SessionStorageTools";
+
+const DEFAULT_ACTIVE_MEMBERSHIP = {
+    memberType: "prime-admin",
+    service: "default",
+};
+
+function setOrganization(parsedName) {
+    const activeMembership = {
+        ...DEFAULT_ACTIVE_MEMBERSHIP,
+        parsedName,
+    };
+
+    window.localStorage.setItem(
+        GLOBAL_STORAGE_KEYS.MEMBERSHIP_STATE,
+        JSON.stringify({
+            activeMembership,
+            memberships: {},
```

Comment:
Rip it out! :D (well, wait for my PR, but then rip it out)
In that vein, is there a better way to assert the shape of this object so IDEs catch it?

Reply:
We could add a type for this and reference that everywhere, but this'd require Cypress to use TypeScript.  It's something we can do, but it just wasn't part of the initial configuration.  I'm a little wary of it because there's a little bit of finagling/casting that needs to happen for Cypress to play nicely with all of our commands, but I'll investigate a little to see how much work it'd be to translate the work here to TypeScript.  If it'd require too much time/too many workaround, I'd say either punt on it or add a ticket to tackle in the future!